### DISCUSSION TITLE
New 1.63 getting trojan warning and my PC keep erasing it.

### DISCUSSION THREAD

**Discussion by BootsOfTango** [LostRuins/koboldcpp#793] [Q&A]
Is it safe to use?

**Comment by LostRuins** [comment]
Yes, it is safe to use. Which file is triggering the false positive?

**Reply by BootsOfTango** [reply]
im sorry i meant i am using cuda

**Reply by LostRuins** [reply]
Hmm, it does seem to be flagged by defender for some reason.

**Reply by BootsOfTango** [reply]
[koboldcpp.exe]
(https://github.com/LostRuins/koboldcpp/releases/download/v1.63/koboldcpp.exe)

**Reply by LostRuins** [reply]
Let me try to rebuild it manually.

**Reply by BootsOfTango** [reply]
ok so imma try to use em anyways, thank you

**Reply by LostRuins** [reply]
it's definitely safe. The cuda one is created by my github pipeline https://github.com/LostRuins/koboldcpp/actions

**Reply by LostRuins** [reply]
rebuilt koboldcpp.exe locally and reuploaded, you can try that

**Reply by Doublespaced1** [reply]
Just tried to download it. Flagged as a virus by Windows and deleted.

**Reply by LostRuins** [reply]
Waiting for Microsoft to unflag the file. Does it still happen?

**Reply by PapaSanzer** [reply]
Sorry to reply to such an old discussion, but this is occurring with AVG now. Same Trojan warning on NoCUDA. That's the only one that flags for me, but seeing this I'm assuming it's an FP. I'll give it an exception

ETA: The specific sig is Win64:Evo-gen [Trj]

**Reply by LostRuins** [reply]
Yeah, false positive. You can trigger the CI to check, all files are identical to the ones built on github.

**Reply by PapaSanzer** [reply]
Yeah everything checked out. I even spun up a VM and used it there first to see if there was a payload. Old habits die hard lol.  I apprecaiate y'alls work on this!
